### PR TITLE
Add ability to turn off complete animation

### PR DESCRIPTION
**Pull request by sgcodigo**
## Status
**READY**

## Description
Allow slide completion animation to be turned off

## Related PRs/Issues
#21 

## Todos
- [ ] Tests
- [ ] Documentation
- [ ] Screenshots

## Feedback Reviews

@cortinico

### CODE REVIEW

File: slidetoact/src/main/res/values/attrs.xml [hunk 1]
```diff
@@ -17,6 +17,7 @@
         <attr name="slider_locked" format="boolean" />
         <attr name="slider_icon" format="reference" />
         <attr name="rotate_icon" format="boolean" />
+        <attr name="animate_complete" format="boolean" />
```
Comment (cortinico) [inline slidetoact/src/main/res/values/attrs.xml]: Can you rename it to `animate_completion`?

File: slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt [hunk 2]
```diff
@@ -436,82 +440,88 @@ class SlideToActView @JvmOverloads constructor (
...
+                }
             }
-        }
 
-        if (mPosition >= mAreaWidth - mAreaHeight) {
-            animSet.playSequentially(marginAnimator, areaAnimator, tickAnimator)
-        } else {
-            animSet.playSequentially(finalPositionAnimator, marginAnimator, areaAnimator, tickAnimator)
-        }
+            if (mPosition >= mAreaWidth - mAreaHeight) {
```
Comment (cortinico) [inline slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt]: What do you think about something like this:

```kotlin
val animators = mutableListOf<Animator>()
if (mPosition < mAreaWidth - mAreaHeight) {
    animators.add(finalPositionAnimator)
}
if (isAnimateComplete){
    animators.add(marginAnimator)
    animators.add(areaAnimator)
    animators.add(tickAnimator)
}
animSet.playSequentially()
```

And for the `startResetAnimation` you probably need something like this:
```
if (isAnimateComplete) {
    animSet.playSequentially(tickAnimator, areaAnimator, positionAnimator, marginAnimator, arrowAnimator)
} else {
    animSet.playSequentially(positionAnimator)
}
```

File: slidetoact/src/main/res/values/attrs.xml (hunk 1 above)
Comment (sgcodigo) [reply slidetoact/src/main/res/values/attrs.xml]: Done.

File: slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt (hunk 2 above)
Comment (sgcodigo) [reply slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt]: Done! :)

**Comment by cortinico** [comment]
Works great! Sorry for the late response :)

**Comment by squeeish** [comment]
Hi @cortinico is this in 0.5.1?

**Comment by cortinico** [comment]
Hey @squeeish, this is not in 0.5.1 will be in 0.6.0

**Pull request merged**